### PR TITLE
Simplify unzip

### PR DESCRIPTION
**Pull request by earforth**

### CODE REVIEW

**Comment by captbaritone** [comment]
This won't work. You can see the failing test for a better idea of why. In the future please include a description of what value you change provides.

Thanks for your interest in Underscore. Feel free to find me in our Gitter channel if you have further questions.

**Comment by craigmichaelmartin** [comment]
lgtm (except for formatting)

File: underscore.js [hunk 1]
```diff
@@ -621,12 +621,9 @@
   // each array's elements on shared indices.
   _.unzip = function(array) {
     var length = array && _.max(array, getLength).length || 0;
-    var result = Array(length);
-
-    for (var index = 0; index < length; index++) {
-      result[index] = _.pluck(array, index);
-    }
-    return result;
+    return _.times(length, function(index) {
```
Comment (jridgewell) [inline underscore.js]: This line has trailing whitespace.

File: underscore.js [hunk 2]
```diff
@@ -621,12 +621,9 @@
...
   _.unzip = function(array) {
     var length = array && _.max(array, getLength).length || 0;
-    var result = Array(length);
-
-    for (var index = 0; index < length; index++) {
-      result[index] = _.pluck(array, index);
-    }
-    return result;
+    return _.times(length, function(index) { 
+        return _.pluck(array, index);
```
Comment (jridgewell) [inline underscore.js]: This line should be indented with 6 spaces.

**Comment by captbaritone** [comment]
I'll cherry pick this commit, clean up the spacing, and open a new PR.

**Comment by jridgewell** [comment]
You could probably push to the PR. https://github.com/blog/2247-improving-collaboration-with-forks

**Comment by captbaritone** [comment]
@jridgewell Oh, that would be a perfect time to use that, but it looks like this commit is not coming from a GitHub fork:

screen shot 2016-12-08 at 9 21 45 am

**Comment by captbaritone** [comment]
Closing in favor of #2634.

**Pull request closed without merging**